config/locales/ja.yml

```
@@ -34,8 +34,8 @@ ja:
     edit: "編集"
     delete: "削除"
     back: "戻る"
-    created_at: "作成日"
     search: "検索"
+    sort: "並べ替え::"
```

「::」は翻訳の中に入れずに、erbのファイルで記載した方がいいと思います！
この記号に翻訳をする必要はないと思うからです！
```suggestion
    sort: "並べ替え"
```